webpack.config.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
   mode: 'development',
   entry: './client/src/index.tsx',
   devtool: 'inline-source-map',
+  watch: true,
```

Comment:
can you think of a way to dynamically set this boolean so you don't have to change it in your development environment vs your production environment?